### ISSUE TITLE
"message" does not appear to be supporting 'typeof Function', contrary to docs.

### ISSUE THREAD

**Issue by thejohnbackes** [amwmedia/plop#45]
According to the Inquirer docs, [message can take either a String or a Function](https://github.com/SBoudrias/Inquirer.js#question):

> message: (String|Function) The question to print. If defined as a function, 
> the first parameter will be the current inquirer session answers.

So, I would expect the following question object to print the previous answers in the prompt:
```
{
      type: 'confirm',
      name: 'confirmCreateComponent',
      message: function (answers) {
        return (
          `Are you sure you would like to create the following component?
          name: <${answers.ComponentName} />
          type: ${answers.componentType}
          location: ${path.join(__dirname, answers.dir)}
          `
        );
      },
```
Instead, it prints the function body in the prompt:
```
? [COMPONENT] function (answers) {
    return (
      `Are you sure you would like to create the following component?
      name: <${answers.ComponentName} />
      type: ${answers.componentType}
      location: ${path.join(__dirname, answers.dir)}
      `
    );
  } (Y/n)
```
Am I misunderstanding the docs?

**Comment by thejohnbackes** [comment]
it is possible this is was fixed in a later version of inquirer. Is there a reason we are using inquirer@^0.1.0 when they are on 1.2.2?

**Comment by amwmedia** [comment]
that was my thought as well. There is a new version of plop that I'm prepping for release that updates to the latest inquirer and changes the core to use the new [node-plop](https://github.com/amwmedia/node-plop) library. I'll test this scenario out on that version and we'll see how it does. plop takes your `prompts` and passes them directly to inquirer, so I think it's pretty likely that it's a version issue.

**Comment by amwmedia** [comment]
the new version of plop is being prepped for release on the [use-node-plop](https://github.com/amwmedia/plop/tree/feature/use-node-plop) branch

**Comment by amwmedia** [comment]
fixed in v1.6.0
